BullsEye/AboutViewController.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  AboutViewController.swift
+//  BullsEye
+//
+//  Created by Tim Lauter on 11/2/16.
+//  Copyright © 2016 timlauter. All rights reserved.
+//
+
+import UIKit
+
+class AboutViewController: UIViewController {
+    @IBOutlet weak var webView: UIWebView!
+    @IBAction func close() {
+        dismiss(animated: true, completion: nil)
+    }
+
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+        
+        if let url = Bundle.main.url(forResource: "BullsEye", withExtension: "html") {
+           
+            if let htmlData = try? Data(contentsOf: url) {
+                let baseURL = URL(fileURLWithPath: Bundle.main.bundlePath)
+                webView.load(htmlData, mimeType: "text/html", textEncodingName: "UTF-8", baseURL: baseURL)
+                
+            }
+            
+        }
+    }
+
+    override func didReceiveMemoryWarning() {
```

Comment:
In general, don't implement a method override unless you're going to do something with the override.